packages/create-react-app/createReactApp.js

```diff
@@ -367,10 +371,11 @@ function run(
   template,
   useYarn,
   usePnp,
-  useTypescript
+  useTypescript,
+  minimal
 ) {
   const packageToInstall = getInstallPackage(version, originalDirectory);
-  const allDependencies = ['react', 'react-dom', packageToInstall];
+  const allDependencies = !minimal ? ['react', 'react-dom', packageToInstall] : ['react@next', 'react-dom@next', packageToInstall];
```

Comment:
Why are you using the `@next` versions of react and react-dom in `minimal` mode? What makes the next versions "minimal"?

Reply:
Why _wouldn't_ you use next?

Reply:
I can name it --gangster mode if you want

Reply:
vote for `--gangster` name

Reply:
I concur

Reply:
There is probably a Tweet by @gaearon somewhere in which he warns about using an `-alpha.x` version in production 😬

Reply:
I'm pretty sure they run it on Facebook prod. I could be wrong.

Reply:
@kenwheeler mostly true, but the difference is that the React team won’t come fix your site if it breaks 😅

I don’t think a minimal template should use unstable APIs, but maybe there could be separate option like `—use-unstable` that lets you experiment with ample warnings

Reply:
@aweary or we could just hold tight until hooks drop and then remove that part.

Reply:
bah, removed it